### PR TITLE
Ticket4877 convert dbchecker python3

### PR DESCRIPTION
**Pull request by JamesKingWork**
Convert the DbChecker to be able to run in Python 3.

Mostly changes to become Pep8 compliant.

Removal of unused imports.

Fix logical error where pvname does not exist (add_sim_records.py function: get_sim_name).

print statement fixes.

### CODE REVIEW

**Comment by JamesKingWork** [comment]
To test run `%PYTHON3% check_db_file.py -d check_db_file_tests > test3.txt` and with python 2. Check that the diff is empty. You can also try this against many of the other dbs such as in EPICS\ISIS.

Also run `%PYTHON3% add_sim_records.py add_sim_records_tests\test_db.db -o test3.db` and for python 2. Check that the diff is empty.

File: db_parser.py [hunk 1]
```diff
@@ -15,76 +15,88 @@ def parse_db(filename):
...
+
+            if inrecord:
+                record_end_match = re.match("}$", line.strip())
+                if not (record_end_match is None):
+                    inrecord = False
                     records[record.name] = record
+                else:
+                    # Look for alias
+                    regex_alias_inside = r'alias\s*\(\s*"([\w_\-\:\[\]' \
+                                         r'<>;$\(\)]+)"\)'
```
Comment (Tom-Willemsen) [inline db_parser.py]: I feel quite strongly that these regexes should not be split over multiple lines - they are hard enough to read as is...

File: db_parser.py [hunk 2]
```diff
@@ -15,76 +15,88 @@ def parse_db(filename):
...
-    reg_record_start = 'record\s*\((\w+),\s*"([\w_\-\:\[\]<>;$\(\)]+)"\)'
+    # This regex matches the EPICS convention for PV names,
+    # e.g. a-z A-Z 0-9 _ - : [ ] < > ;
+    # The ISIS convention uses a reduced set (a-z A-Z 0-9 _ :)
+    # but this is not relevant at this level
+    reg_record_start = r'record\s*\((\w+),\s*"([\w_\-\:\[\]<>;$\(\)]+)"\)'
     # Aliases look like alias("RECORDNAME","ALIASNAME")
-    reg_alias_alone = 'alias\s*\(\s*"([\w_\-\:\[\]<>;$\(\)]+)",\s*"([\w_\-\:\[\]<>;$\(\)]+)"\)'
+    reg_alias_alone = r'alias\s*\(\s*"([\w_\-\:\[\]<>;$\(\)]+)",' \
+                      r'\s*"([\w_\-\:\[\]<>;$\(\)]+)"\)'
```
Comment (Tom-Willemsen) [inline db_parser.py]: Don't split regexes over multiple lines - even if it goes over the style convention's limit readability wins over convention here...

**Review by Tom-Willemsen — CHANGES_REQUESTED** [review]

File: db_parser.py (hunk 1 above)
Comment (JamesKingWork) [reply db_parser.py]: Fair enough, changed

**Review by Tom-Willemsen — APPROVED** [review]

**Pull request merged**